web-client/src/components/UserDataForm/UserDataForm.tsx

```diff
@@ -1,5 +1,6 @@
 import { Button, Checkbox, Form, Input, Typography } from 'antd';
-import React from 'react';
+import _ from 'lodash';
```

Comment:
Here you can import `words` from lodash like so:

`import words from 'lodash/words';`

This will reduce bundle size as we not importing lodash's full package.

Reply:
Can I import like this? `import { words } from 'lodash';`